client/src/components/molecules/RequestsTable.tsx

```diff
@@ -38,19 +39,17 @@ const RequestsTable: FunctionComponent<Props> = (props: Props) => {
         }
     `;
 
-    const [requests, setRequests] = useState(props.requests.filter((request) => request.deleted === false));
-
     useEffect(() => {
         const nonFulfilledRequests = requests.filter((request) => {
             if (request !== undefined) {
-                if (request.fulfilled === false) {
+                if (request.fulfilled == null) {
```

Comment:
Also curious about this change too — is there a circumstance in which `request.fulfilled` wouldn't be a boolean?

Reply:
That's a typo, should be fulfilledAt. Updated